### PR TITLE
Packages: Update Hoek dependencies.

### PR DESCRIPTION
**Pull request by kwight**
This PR updates the repo to Webpack 3. `webpack-dev-server` also gets bumped to 2.11.2.

**Testing**

1. Blow away `node-modules` and do a fresh `npm install`.
2. Smoke test as much of the app as possible; there should be no related errors or regressions.
3. There should be no references to the `hoek` package in `package-lock.json`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,5 +70,11 @@
     "webpack-dev-server": "2.4.2",
     "wpcom": "5.3.0",
     "wpcom-proxy-request": "3.1.0"
+  },
+  "dependencies": {
+    "boom": "^7.2.0",
+    "cryptiles": "^4.1.1",
+    "fsevents": "^1.2.4",
+    "hawk": "^7.0.7"
```
Comment (dmsnell) [inline package.json]: these probably belong above in `devDependencies` no?

also, why are we _adding_ these? I think we want to keep dependencies-of-dependencies out of `package.json` and update instead the primary dependencies which are pulling in the vulnerable version of `hoek`

Comment (kwight) [reply package.json]: Wuuuuuuttt??!! Haha, I never even noticed I was adding these to `package.json` – I verified what I wanted to see in the package lock file and pushed 🙃Thanks, yes, I agree it should be just updating the dependencies themselves, will sort it out on Monday.

**Comment by kwight** [comment]
@dmsnell I think this should take care of it now.

**Comment by kwight** [comment]
> if you regenerated the package-lock.json file from scratch then :shipit:

Yep; tested by confirming the changes worked fine, then blew away both `node_modules` and `package-lock.json`, then did a fresh install and verified again. Thanks for the review!

**Pull request merged**